### PR TITLE
Dispatch event on preview request

### PR DESCRIPTION
**Pull request by rullzer**
Fixes: #73

@LukasReschke as discussed.

### CODE REVIEW

File: lib/public/IPreview.php [hunk 1]
```diff
@@ -43,6 +43,8 @@
  */
 interface IPreview {
 
+	const EVENT = self::class . ':' . 'PREVIEWREQUESTED';
```
Comment (nickvergessen) [inline lib/public/IPreview.php]: stop shouting, use CamelCase?

Comment (nickvergessen) [inline lib/public/IPreview.php]: Also add a `@since`

Comment (rullzer) [inline lib/public/IPreview.php]: Done

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
@nickvergessen @icewind1991 Please review ;)

**Comment by icewind1991** [comment]
Looks good :+1:

**Pull request merged**